stacks/_templates/postgresql-timescaledb.yaml

```diff
@@ -5,6 +5,8 @@ repo:
   url: https://charts.timescale.com/
 version: 0.33.1
 options:
+  commonLabels:
```

Comment:
commonlabels throws an exception for me with timescaledb
```suggestion
```

Reply:
Changed in 0f0b04a. This *should* fix the error.

Side note: The official Helm chart for timescaledb was archived on [2023-01-18](https://github.com/timescale/helm-charts) and it seems it was unmaintained for a while already: https://github.com/timescale/helm-charts/issues/596. I can create a tracking issue to find a suitable replacement.

Reply:
This doesn't seem to label anything on the pod but the error has gone.